commands/bmc.vegman

```diff
@@ -353,3 +353,86 @@ function cmd_reboot {
   echo "Reboot the BMC..."
   reboot ${force}
 }
+
+# @doc cmd_remoteimage
+# Access to the remote side file
+function cmd_remoteimage {
+  subcommand "$@"
+}
+
+# @sudo cmd_remoteimage_connect admin
```

Comment:
The `operator` role allows their members to manage the host.
Shouldn't we also allow these commands for the operators?

Reply:
At the [CLI Interface Proposal](https://c.yadro.com/pages/viewpage.action?spaceKey=LKB&title=VEGMAN+CLI+Proposal), the VirtualMedia command is allowed only for admin.
I have followed these recommendations, but of course, I think we might spread user access from the operator group.

Reply:
Leave it as is so far